src/components/Main.js

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from 'react'
+import {FiBookmark} from  "react-icons/fi";
+import {CgCloseO} from "react-icons/cg";
+
+
+export default function Main() {
+
+   let [OpenModal, setOpenModal] = useState(true);
+   function isOpenModal () { 
+      setOpenModal(OpenModal = !OpenModal);
```

Comment:
лучше не присваивать в стейт напрямую - именно для этого и сеть функция `setOpenModal` - она присваивает новое значение в `OpenModal`
```suggestion
      setOpenModal(!OpenModal);
```